policyengine_us/variables/gov/states/nc/tax/income/nc_taxable_income.py

```diff
@@ -9,5 +9,5 @@ class nc_taxable_income(Variable):
     definition_period = YEAR
     defined_for = StateCode.NC
 
-    adds = ["adjusted_gross_income", "nc_additions"]
+    adds = ["adjusted_gross_income", "nc_additions", "nc_use_tax"]
```

Comment:
remove from here, we should rather include the use tax in `nc_income_tax` as it is applied after the taxable income is calculated